_redirects

```diff
@@ -0,0 +1 @@
+/*    /index.html   200
```

Comment:
Easy peasy! Consider adding a trailing `\n`, otherwise this looks 👍

Reply:
Where are you Prettier